packages/plus/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -34,27 +119,57 @@ export const CodeEditor = ({
   height,
   readOnly,
   autoCompletion,
+  disabled = false,
+  label,
+  copyButton,
+  id,
+  helper,
+  labelDescription,
+  'aria-label': ariaLabel,
+  'data-testid': dataTestId,
+  className,
 }: CodeEditorProps) => (
-  <div>
-    {title ? (
-      <StyledText as="h3" variant="headingSmall">
-        {title}
-      </StyledText>
+  <StyledStack gap={0.5} data-disabled={disabled}>
+    {label || title ? (
+      <Label labelDescription={labelDescription} disabled={disabled}>
+        {label ?? title}
+      </Label>
+    ) : null}
```

Comment:
Why is the label light gray in disabled mode? doesn't seem we have that anywhere else
![Screenshot 2025-05-05 at 09 18 00](https://github.com/user-attachments/assets/7805d557-a29c-44c2-80e4-6e9232faeddd)